src/common/OpQueue.h

```diff
@@ -37,10 +37,15 @@ class OpQueue {
   public:
     // How many Ops are in the queue
     virtual unsigned length() const = 0;
-    // Ops will be removed f evaluates to true, f may have sideeffects
+    // Ops will be removed f evaluates to true, f may have side effects.
+    // Must visit operations from back to front, due to its use with
+    // OSD::SharedOpQueue::Pred .
     virtual void remove_by_filter(
```

Comment:
Note that hte OSD no longer needs or uses this remove_by_filter thing (as of 3cc48278bf0ee5c9535d04b60a661f988c50063b), unless it's been reintroduced by this series...